packages/dxf-deserializer/test/test-dxf-versions.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs')
+const path = require('path')
+const test = require('ava')
+const { CSG, CAG } = require('@jscad/csg')
+
+import {nearlyEqual} from '../../../test/helpers/nearlyEqual'
+
+const { deserialize } = require( '../index' )
+
+const samples = path.resolve('../../node_modules/@jscad/sample-files')
+
+//
+// Test suite for DXF deserialization (import)
+//
+test('ASCII DXF R13 to Object Conversion', t => {
```

Comment:
eek , I always forget how many different versions of the format there are, this is huge work , kudos !

Reply:
As many as there are years from 1901... ha ha ha